rebar.config

```diff
@@ -119,6 +119,10 @@
                    {copy, "data/aecore/.minerva/accounts.json", "data/aecore/.minerva/accounts.json"},
                    {copy, "data/aecore/.minerva/accounts_uat.json", "data/aecore/.minerva/accounts_uat.json"},
                    {copy, "data/aecore/.minerva/accounts_test.json", "data/aecore/.minerva/accounts_test.json"},
+                   {mkdir, "data/aecore/.fortuna"},
```

Comment:
If this is here, then `ci/appveyor/package.bat` shall be amended too? cc @tolbrino

Reply:
Yes.

Reply:
OK, will do... Didn't even know there was such a file... Why not use the marvelous rebar for this? 😂

Reply:
Don't make me do it O_O